dockerfiles/postgres/Dockerfile

```diff
@@ -6,7 +6,7 @@
 # SPDX-License-Identifier: EPL-2.0
 #
 
-FROM centos/postgresql-96-centos7
+FROM registry.access.redhat.com/rhscl/postgresql-10-rhel7
```

Comment:
We need to make sure that the upgrade for existed installation from postgresql 9.6 to 10 is working.
Have you tested it?

Reply:
does it work with `docker.io/postgres:9.6.17-alpine` or `docker.io/postgres:9.6.17` (debian based)
 (no bump of the postgres version) ?

Reply:
@skabashnyuk I must confess that this use case skipped from my mind. So if updating postgresql version is causing other overheads, I'll revert the changes for `9.6.17` as @benoitf has suggested.

@benoitf I am able to build `che-postgres` docker image using both variants you suggested i.e. `docker.io/postgres:9.6.17-alpine` and `docker.io/postgres:9.6.17`. Let me know your preference for the base image. Will update the PR with same.

Reply:
@shahidhs-ibm there are quite a few scripts that make PostgreSQL installation possible https://github.com/eclipse/che/tree/master/dockerfiles/postgres can you check if they compatible with `docker.io/postgres:9.6.17`?
CC @tolusha

Reply:
@skabashnyuk The `che-postgres` docker image built with `docker.io/postgres:9.6.17-alpine` is not compatible with the [scripts](https://github.com/eclipse/che/tree/master/dockerfiles/postgres). The [centos](https://github.com/sclorg/postgresql-container/tree/generated/9.6) Vs [alpine](https://github.com/docker-library/postgres/tree/master/9.6) postgres images are built with different methodology which makes them incompatible. 

This best possible approach is to extend `postgresql-96` docker image support to s390x. We have raised an [issue](https://github.com/sclorg/postgresql-container/issues/376) for this.

Reply:
@skabashnyuk I came across [registry.redhat.io/rhel8/postgresql-96](https://catalog.redhat.com/software/containers/detail/5ba0ad1f5a134643ef2eeb9d?architecture=s390x&container-tabs=overview) image which can serve the purpose. This is a RH subscription image.

Earlier I thought that subscription image we may not use but I came across this [commit](https://github.com/eclipse/che-machine-exec/commit/1ce224eb15cb339303fd1fecdafaad6d5f76cb0c) in `che-machine-exec` which uses RH subscription image. 

Could you please confirm if it is OK to use this image?

Reply:
СС @tolusha @nickboldt

Reply:
@tolusha @nickboldt @benoitf Your updates?

Reply:
The best way if we find (build ?) `centos/postgresql-96-centos7` image for `s390x` platform
Otherwise we have to perform a lot of tests to be sure the previous deployments work well.
BTW codeready uses `registry.redhat.io/rhscl/postgresql-96-rhel7:1-47`

Reply:
@tolusha Thanks for the update. Yes the best way will be to get the image from centos but we haven't got any response from community on issue we raised for code contribution. The `registry.redhat.io/rhscl/postgresql-96-rhel7:1-47` which is used by CRW is unfortunately not multi-arch, so we don't have many options left other than using `registry.redhat.io/rhel8/postgresql-96` or adding multi-arch support to `centos/postgresql-96-centos7`.

@nickboldt Could you please provide your inputs on using RH subscription image for postgres? Will it be OK?

Reply:
@nickboldt Any update?

Reply:
Note that `registry.redhat.io/rhscl/postgresql-96-rhel7:1-47` is single arch, amd64.

We switched to `registry.redhat.io/rhel8/postgresql-96:1` to get support for Z and Power (it also includes arm).